nominees/housing-vacancy-bot.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Housing Vacancy Bot",
+  "description": "A Telegram bot that allows housing officials/owners to report vacancy to a central source and allows health works/government officials to consume that information.",
+  "website": "https://github.com/africa-covid-19-response-toolkit/housing-vacancy-bot",
```

Comment:
@318anushka kindly update the website field to point to a websiteURL rather than Github repo. Thanks.